include/ccf/crypto/cose_signer.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
```

Comment:
Unless/until there is a clear user requirement to expose this, let's keep internal (i.e. src, not include/) to avoid expanding our public API unnecessarily, and preserve our ability to make changes quickly in the future. We probably want to add protected header parameters at COSESigner construction time for example.

Reply:
Removed this